### ISSUE TITLE
[Bug]: Running taq deploy [contractName] return [object Object] instead of contract with its hash

### ISSUE THREAD

**Issue by sinapsist** [ecadlabs/taqueria#387]
## To Do:

- [x] Fix bug
- [x] Vaildate and improve tests
- [x] #493

### What happened?

After the latest release to 0.1.1 the test that checks "taq deploy [contractName]" starts failing due to the issue with the command output

### What should have happened?

The command output should be in stdout and should include table with contract name and contract hash

### Steps to reproduce the bug

1. Run taq deploy [contractName] (e. g. hello-tacos.tz) and observe result

### Relevant log output

```shell
*** START Call to @taqueria/plugin-taquito ***
    node \
    /Users/ivanromanov/workspace/ecadlabs/taqueria/tests/e2e/auto-test-taquito-plugin/node_modules/@taqueria/plugin-taquito/index.js \
    --taqRun \
    proxy \
    --i18n \
    '{}' \
    --config \
    '{"language":"en","plugins":[{"name":"@taqueria/plugin-taquito","type":"npm"}],"contractsDir":"contracts","testsDir":"tests","artifactsDir":"artifacts","environment":{"default":"development","development":{"networks":[],"sandboxes":["local"],"storage":{}},"test":{"networks":["hangzhounet"],"sandboxes":[],"storage":{"hello-tacos.tz":"100"}}},"sandbox":{"local":{"accounts":{"default":"bob","bob":{"initialBalance":"3000000000"},"alice":{"initialBalance":"2000000000"},"john":{"initialBalance":"4000000000"},"jane":{"initialBalance":"5000000000"},"joe":{"initialBalance":"1000000000"}},"label":"Local Tezos Sandbox","protocol":"PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx","rpcUrl":"http://localhost:20000"}},"network":{"hangzhounet":{"label":"hangzhounet","rpcUrl":"https://hangzhounet.api.tez.ie","protocol":"PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx","faucet":{"pkh":"tz1UjBsGr8Tkogbz8fcySP6x77S5TKBvs99t","mnemonic":["bone","copper","early","light","shallow","sense","burden","civil","knee","knife","excuse","noise","enable","piano","loyal"],"email":"tasbcxmi.wbdftfiw@teztnets.xyz","password":"sz2Trjg3Z5","amount":"55579807681","activation_code":"cce0f1f37d4eb45b5ac0cdf755825abd5fd53b10"}}},"configFile":{"value":"/Users/ivanromanov/workspace/ecadlabs/taqueria/tests/e2e/auto-test-taquito-plugin/.taq/config.json"},"configDir":{"value":"./.taq"},"projectDir":{"value":"/Users/ivanromanov/workspace/ecadlabs/taqueria/tests/e2e/auto-test-taquito-plugin"},"hash":{"value":"077f8b1b4a2ddd5f8e0cf0192f3c5a936a4bd01ab5aa1e795006cc25bb0aeb8f"}}' \
    --envVars \
    '{}' \
    --command \
    deploy,hello-tacos.tz \
    --setBuild \
    'taquito-plugin-tests-1645669899' \
    --setVersion \
    'dev:taquito-plugin-tests' \
    --e \
    'test' \
    --env \
    'test' \
    --logPluginCalls \
    'true' \
    --disableState \
    'false' \
    --maxConcurrency \
    '10' \
    --debug \
    'false' \
    --p \
    './' \
    --projectDir \
    '/Users/ivanromanov/workspace/ecadlabs/taqueria/tests/e2e/auto-test-taquito-plugin' \
    --d \
    './.taq' \
    --configDir \
    './.taq' \
    --fromVsCode \
    'false' \
    --contract \
    'hello-tacos.tz' \
    --scaffoldUrl \
    '' \
    --scaffoldProjectDir \
    '/Users/ivanromanov/workspace/ecadlabs/taqueria/tests/e2e/auto-test-taquito-plugin' \
    --task \
    'deploy'
    *** END of call to @taqueria/plugin-taquito ***
    ┌─────────────────┐
    │ 0               │
    ├─────────────────┤
    │ [object Object] │
    └─────────────────┘
```

### Screenshots and images

Screenshot and images

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by russellmorton** [comment]
Specifying a contract name to deploy triggers error.

**Comment by mweichert** [comment]
Fixed in #465.
